src/components/pages/Transaction/TransactionReview.tsx

```diff
@@ -113,12 +119,11 @@ function TransactionReview() {
         owner: walletAddress,
         processId: ioProcessId,
         dispatch: dispatchTransactionState,
-        signer: wallet?.arconnectSigner,
+        signer: wallet?.contractSigner,
         ao: aoClient,
         scheduler: aoNetwork.SCHEDULER,
       });
-
-      navigate('/transaction/complete');
+      // navigate('/transaction/complete');
```

Comment:
can remove - or if not put a comment as to the previous issue with ref to the stack overflow posts.